flow/designs/asap7/mock-array/Element/config.mk

```diff
@@ -26,13 +26,9 @@ export IO_CONSTRAINTS         = designs/asap7/mock-array/Element/io.tcl
 
 export PDN_TCL                = $(FLOW_HOME)/platforms/asap7/openRoad/pdn/BLOCK_grid_strategy.tcl
 
-# If this design isn't quickly done in detailed routing, something is wrong.
-# At time of adding this option, only 3 iterations were needed for 0
-# violations.
-export DETAILED_ROUTE_ARGS=-bottom_routing_layer M2 -top_routing_layer M5 -save_guide_updates -verbose 1 -droute_end_iter 10
+# Detailed routing should be easy, limit iterations
+export DETAILED_ROUTE_END_ITERATION = 6
```

Comment:
If it is easy then it will finish naturally without needing a limit.

Reply:
True, and this limit is to ensure that it stays that way...